elastic/beats
# Add crawler  metrics into the stats metricset for Enterprise Search

## What does this PR do?

This PR adds more metrics to the stats metricset for Enterprise Search. The new metrics have been added to Enterprise Search in https://github.com/elastic/ent-search/pull/5192 and will be available in 8.0.0.

## Why is it important?

Users need these new metrics to be able to reason about the health of their crawler deployments.

## Checklist


## Author's Checklist

- [x] Tested it by running the metricbeat module locally against the most recent version of Enterprise Search.

## How to test this PR locally

* Run the most recent version of Enterprise Search
* Run metricbeat with the enterprisesearch module enabled and pointed at your local instance
* Check metrics ingested into Elasticsearch

## Related issues

- Relates to https://github.com/elastic/app-search-team/issues/2071
